core/layouts/two_column/layout.css

```diff
@@ -1,6 +1,6 @@
 /**
  * @file
- * CSS for the three-three-four layout.
```

Comment:
I came across this little mistake and fixed it. Should I have created a new issue instead?